eth/filters/api.go

```diff
@@ -334,6 +338,9 @@ func (api *FilterAPI) NewFilter(crit FilterCriteria) (rpc.ID, error) {
 
 // GetLogs returns logs matching the given argument that are stored within the state.
 func (api *FilterAPI) GetLogs(ctx context.Context, crit FilterCriteria) ([]*types.Log, error) {
+	if len(crit.Topics) > maxTopics {
+		return nil, errInvalidTopic
```

Comment:
Perhaps use something more similar to reth? 
```suggestion
		return nil, errors.New("exceeded maximum topics")
```
and if so, also the other place below